src/queries/category/edit.ts

```diff
@@ -0,0 +1,17 @@
+import Category from '../../models/Category'
+import { type ICategory } from '../../interfaces/models'
+
+const editCategory = async (categoryId: number, title: string): Promise<ICategory | null> => {
+  const category = await Category.findByPk(categoryId)
+
+  if (category != null) {
+    category.title = title
+    await category.save()
```

Comment:
I'm not sure if I got this right, but if you are trying to update the category title maybe you should take a look at this link
https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#simple-update-queries